javascript/javascript2/week2/flight-fullname.js

```diff
@@ -0,0 +1,11 @@
+function getFullname (firstname, surname, useFormalName) {
+  if (useFormalName === true) {
+    return "Lord " + firstname + " " + surname
+  } else {
+    return firstname + " " + surname
+  }
+};
```

Comment:
You need to add two more if conditions to check whether firstName and lastName are not null. Read the problem description clearly and try to implement all requirements. 
see the below text given in problem description
"Remember to consider someone calling the function with an empty string as firstname and lastname"